sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure TrustedSigning management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
Describe your service here

Reply:
I have added the description in readme.md